model/QtiCreatorClientConfigRegistry.php

```diff
@@ -30,7 +30,7 @@
  */
 class QtiCreatorClientConfigRegistry extends ClientLibConfigRegistry
 {
-    const CREATOR = "taoQtiItem/controller/creator/index";
+    public const CREATOR = "taoQtiItem/controller/creator/index";
```

Comment:
suggestion: can we make it private if its used only here?

Reply:
No, it is used somewhere